js/mainjs/initSiad.js

```diff
@@ -132,7 +132,7 @@ function checkSiadPath() {
 			checkSiadPath();
 		} else if (selected === 0) {
 			var lastIndex = siadPath.lastIndexOf('/');
-			config.siad.command = siadPath.substring(lastIndex);
+			config.siad.fileName = siadPath.substring(lastIndex);
```

Comment:
Oh I renamed `siad.command` to `siad.fileName` in `sia.js` of the node wrapper and forgot to reflect that change here.